### ISSUE TITLE
Flaresolverr default settings

### ISSUE THREAD

**Issue by adgriff2** [tteck/Proxmox#3341]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

The https://tteck.github.io/Proxmox/#flaresolverr-lxc site list Flaresolverr default settings as:
Default Settings: 512MiB RAM - 2GB Storage - 1vCPU

Script defaults to 2048MiB RAM - 4GB Storage - 2vCPU

If you use the default settings listed on the website, install fails unable to extract chrome install. Likely just need to update site default settings label.

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

**Comment by tteck** [comment]
That is the build settings, once finished it sets the default settings.
https://github.com/tteck/Proxmox/blob/186f186e8b6c8d706adc8a110646f93147e35448/ct/flaresolverr.sh#L79-L82

I'll check the chrome install.

**Comment by tteck** [comment]
🏁 The script runs without any problems from this end.

```
    ________               _____       __                    
   / ____/ /___ _________ / ___/____  / /   _____  __________
  / /_  / / __ `/ ___/ _ \\__ \/ __ \/ / | / / _ \/ ___/ ___/
 / __/ / / /_/ / /  /  __/__/ / /_/ / /| |/ /  __/ /  / /    
/_/   /_/\__,_/_/   \___/____/\____/_/ |___/\___/_/  /_/     

Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: flaresolverr
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a FlareSolverr LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.114.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Chrome
 ✓ Installed FlareSolverr
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Set Container to Normal Resources
 ✓ Completed Successfully!

FlareSolverr should be reachable by going to the following URL.
         http://192.168.86.250:8191 
```
🧐 As this issue is unrelated to the script, it is considered closed.

**Comment by rmarineau** [comment]
Wanted to note that I encountered an issue when updating the flaresolverr container that sounded similar to this. In my case, while doing an update to the container, all packages updated successfully with the exception of chrome. This is the error I encountered repeatedly:

`cannot copy extracted data for './opt/google/chrome/chrome' to '/opt/google/chrome/chrome.dpkg-new': unexpected end of file or stream`

After many attempts at updating, I was able to get the update to successfully complete only after increasing the container RAM to 1GB.

Edit: Just to add, I was also using Debian 12 with default settings until increasing the ram.
